### PR TITLE
flatcar/v1_2_exp: Add clevis support

### PR DESCRIPTION
**Pull request by simoncampion**
This PR updates `flatcar/v1_2_exp` to reflect Flatcar can handle disk encryption with clevis, since [this commit](https://github.com/flatcar/scripts/pull/1560).

### CODE REVIEW

File: config/flatcar/v1_2_exp/translate.go [hunk 1]
```diff
@@ -22,15 +22,9 @@ import (
...
-var (
-	fieldFilters = cutil.NewFilters(types.Config{}, cutil.FilterMap{
-		"storage.luks.clevis": common.ErrClevisSupport,
-	})
-)
-
 // Return FieldFilters for this spec.
 func (c Config) FieldFilters() *cutil.FieldFilters {
-	return &fieldFilters
+	return &cutil.FieldFilters{}
```
Comment (pothos) [inline config/flatcar/v1_2_exp/translate.go:27]:
```suggestion
	return nil
```
Thanks for the PR! The only thing I would see is that the other `translate.go` files use `nil` here

Comment (simoncampion) [reply config/flatcar/v1_2_exp/translate.go]: Changed it to `nil` here. (This also required a tiny change to the test logic, handling the case of `FieldFilters()` returning `nil`.)

**Review by prestist — COMMENTED** [review]
Awesome, thank you for the update! 
The changes lgtm, but could you add a comment in the release notes please :)

**Comment by simoncampion** [comment]
@prestist Thanks! I added an entry in the release notes and ran `./generate`.

**Review by prestist — APPROVED** [review]
Thank you for working on this. Overall LGTM!
